darts/timeseries.py

```diff
@@ -775,6 +776,12 @@ def from_group_dataframe(
         List[TimeSeries]
             A list containing a univariate or multivariate deterministic TimeSeries per group in the DataFrame.
         """
+        if time_col is None and df.index.is_monotonic_increasing:
+            logger.warning(
+                "UserWarning: `time_col` was not set and `df` has a monotonically increasing (time) index. You can "
+                "ignore this warning if the index represents the actual index of each individual time series group."
```

Comment:
I would maybe add something between parentheses saying that a monotonically increasing time index in this context means that the groups have non-overlapping (time) index.

Reply:
👍